s3/pkg/meta/db/drivers/tidb/gc.go

```diff
@@ -94,21 +91,20 @@ func (t *TidbClient) ListGcObjs(ctx context.Context, offset, limit int) (objs []
 	}
 	defer rows.Close()
 
-	var iversion uint64
 	for rows.Next() {
 		obj := &Object{Object: &pb.Object{}}
 
 		err = rows.Scan(
 			&obj.BucketName,
 			&obj.ObjectKey,
-			&iversion,
+			&obj.VersionId,
 			&obj.Location,
 			&obj.ObjectId,
 			&obj.StorageMeta,
 		)
-		timestamp := math.MaxUint64 - iversion
+		/*timestamp := math.MaxUint64 - iversion
```

Comment:
@sunfch @hopkings2008 Please check this part, is there any problem to change like this?